ministryofjustice/analytics-platform-control-panel
# Enable normal user to update their own details

## What

* Added `IsSelf` permission mixin, to allow normal user to access and modify their own details
* Added `verified_email` flag to User model to record whether a user has verified their email address

## How to review

1. `docker-compose up`
2. `docker-compose exec app python3 manage.py shell` 
3. Make yourself a normal user, eg:
    ```python
    from control_panel_api.models import User
    u = User.objects.get(username='your username')
    u.is_superuser = False
    u.save()
    ```
4. Browse to http://localhost:8000/users/{your_auth0_id}/
5. See that you get a response, and not an error

See [Trello #453](https://trello.com/c/REq10r8F)
